vm/safepoints.cpp

```diff
@@ -35,11 +35,18 @@ void factor_vm::handle_safepoint(cell pc) {
   faulting_p = false;
 
   if (atomic::load(&safepoint_fep_p)) {
+    atomic::store(&safepoint_fep_p, false);
```

Comment:
@mrjbq7: an exception is being thrown below, so the code at the bottom will never be executed.